packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -357,7 +359,15 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 		});
 	}
 
-	async deleteExecutions(
+	async softDeleteExecution(executionId: string) {
+		await this.update({ id: executionId }, { deletedAt: Date.now() });
+	}
+
+	async softDeleteExecutions(executionIds: string[]) {
+		await this.update({ id: In(executionIds) }, { deletedAt: Date.now() });
+	}
```

Comment:
Do we need both? Can we remove `softDeleteExecution` and use the plural version always? Or use a unified version acccepting `string | string[]` and check with `Array.isArray()`? And if we unify, maybe simply `softDelete`? The deletion target is clear from context and this'd be consistent with other repositories and less verbose.